games/ludo/master.py

```diff
@@ -33,7 +32,9 @@ class LudoGameMaster(GameMaster):
     def __init__(
         self,
         experiment: dict[str: dict],
-        player_models: list[Model]
+        player_models: list[Model],
+        chain_of_thought: bool,
+        reprompting: bool
```

Comment:
Maybe we should add default values for CoT and Reprompting. I'd say false since it would be the most basic version.